charts/templates/configmap-scripts.yaml

```diff
@@ -839,6 +841,8 @@ data:
         configure_tls
     fi
 
+    touch /tmp/marklogic/ready
```

Comment:
I'm not sure the startup probe is needed on our case.
Startup probe are mostly used for slow starting container.
In our case the Container will be marked as Running only if the post-start hook is OK.
As well our liveness probe does not kill the container.